schema/system/entity.json

```diff
@@ -27,7 +27,8 @@
         },
         "schemaVersion": {
             "description": "entity's schema version. Used to distinct between different schemas.",
-            "type": "string"
+            "type": "string",
+            "default": "0.2.0"
```

Comment:
1. We should probably separate to a `schemaVersion` file and include from there.
2. Should we update to reflect on the current date - `2022.8.16`?